### ISSUE TITLE
Add "r3" etc. aliases to the pt_regs_offset table

### ISSUE THREAD

**Issue by mpe** [linuxppc/linux#10]
That way when defining `kprobe_events` we can use `r3` etc. rather than currently where you have to use `gpr3`, which is a little odd on powerpc.

**Comment by mpe** [comment]
For example, currently we can do:

```
$ su -
$ echo "p:probe/sys_fchownat .sys_fchownat %gpr3:s32 +0(%gpr4):string %gpr5:s32 %gpr6:s32 %gpr7:s32" > /sys/kernel/debug/tracing/kprobe_events
$ echo 1 > /sys/kernel/debug/tracing/events/probe/sys_fchownat/enable
$ touch /tmp/foo
$ chown root /tmp/foo
$ echo 0 > /sys/kernel/debug/tracing/events/enable
$ cat /sys/kernel/debug/tracing/trace
# tracer: nop
#
# entries-in-buffer/entries-written: 1/1   #P:8
#
#                              _-----=> irqs-off
#                             / _----=> need-resched
#                            | / _---=> hardirq/softirq
#                            || / _--=> preempt-depth
#                            ||| /     delay
#           TASK-PID   CPU#  ||||    TIMESTAMP  FUNCTION
#              | |       |   ||||       |         |
           chown-5040  [001] d... 24800.047211: sys_fchownat: (.SyS_fchownat+0x0/0x190) arg1=-100 arg2="foo" arg3=0 arg4=-1 arg5=0
```

Instead we'd like to be able to use eg:

```
$ echo "p:probe/sys_fchownat .sys_fchownat %r3:s32 +0(%r4):string %r5:s32 %r6:s32 %r7:s32" > /sys/kernel/debug/tracing/kprobe_events
```

**Comment by mpe** [comment]
The reverse transformation, ie. from a register offset to a name, is implemented by `regs_query_register_name()`. Currently this is unused.

But it would still be good if it returned the powerpc style names, so that if it is ever used it uses those names.

**Comment by mpe** [comment]
Fixed in 343c3327c12b13655192983007de94bd44fd7941.
